sklearn/tests/test_multiclass.py

```diff
@@ -921,3 +922,108 @@ def test_ovo_consistent_binary_classification():
     ovo.fit(X, y)
 
     assert_array_equal(clf.predict(X), ovo.predict(X))
+
+
```

Comment:
putting a beginning and end sections for the routing tests would be nice, eg:

https://github.com/scikit-learn/scikit-learn/blob/5dbb8f52dc5c9902503b0b686f25916321a0ebbd/sklearn/tests/test_pipeline.py#L1693-L1694